specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,192 @@
+Badge Notifications in Windows App SDK
+===
+
+This is the spec for proposal: [Issue #138](https://github.com/microsoft/WindowsAppSDK/issues/138)
+
+# Background
+
+A notification badge conveys summary or status information specific to your app. They can be numeric (1-99) or one of a set of system-provided glyphs. Examples of information best conveyed through a badge include network connection status in an online game, user status in a messaging app, number of unread mails in a mail app, and number of new posts in a social media app. 
+
+Example of a numeric badge in Taskbar (One new message in Whatsapp): 
+
+![Screenshot](NumericBadgeExample.png)
+
+Example of a system-provided glyph in taskbar (New message in Outlook): 
+
+![Screenshot](GlyphBadgeExample.png)
+ 
+This Badge notification functionality is missing in WinAppSDK, which not only blocks apps like WhatsApp to move to WinAppSDK but also stops them from availing new features and goodness that we are building in WinAppSDK. The scope of this document is to fill that gap and provide badge notification capability in WinAppSDK as well.   
+
+# Description
+
+Badge notifications provide users with a visual indicator of new or important content in your application. This feature enhances user engagement by allowing them to quickly see updates without needing to open the app. Badge notifications are already supported for UWP apps. This initiative is to make badge notifications work in WindowsAppSdk.
+
+For more details see:
+
+-   [Badge Notification WinRT APIs](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/badges)
+    Defines all the API constructs that we have for Badge Notifications in WinRT today.
+
+# Examples
+
+## Create a numeric badge (c++)
+
+ ```c++
+private void setBadgeNumber(uint32_t num)
+{
+    winrt::BadgeNotification badgeNotification(num);
+    winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Default();
+    manager.UpdateBadge(badgeNotification);
+}
+```
+
+## Create a Glyph badge (c++)
+
+ ```c++
+private void updateBadgeGlyph()
+{
+    winrt::BadgeGlyphValue badgeGlyphValue = winrt::BadgeGlyphValue::alert
+    winrt::BadgeNotification badgeNotification(badgeGlyphValue);
+    winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Default();
+    manager.UpdateBadge(badgeNotification);
+}
+```
+
+## Clear a badge
+
+ ```c++
+private void clearBadge()
+{
+    winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Default();
+    manager.ClearBadge();
```

Comment:
The verb methods on the `BadgeNotificationManager` don't need to include "Badge" in their method names. Instead, call this `Clear()` and then `UpdateBadge` becomes `Update`, etc.

Reply:
Done